docs/video/obs/output.md

```diff
@@ -179,7 +179,7 @@ You're recommended to use the [latest](https://github.com/obsproject/obs-studio/
 
     ##### Description
 
-    Similar to [NVIDIA Shadowplay](https://www.nvidia.com/en-us/geforce/geforce-experience/shadowplay/), this captures using your recording settings and keeps in RAM only the last <kbd>X</kbd> seconds of what was captured and at any point you can press a hotkey to save it as a video file.
```

Comment:
i think this is important to mention as people don't always make the connection that it uses the recording settings